Add a note for "x-auth-type" extention. (#121)
wso2/docs-mg: en/docs/concepts/openapi-extensions.md

| `x-wso2-mutual-ssl`               | Enable mutual SSL for an API (With optional and mandatory keywords as values)                                          | Optional → API level only      |
| `x-wso2-owner`                    | Specifies the owner of the API. It is used to view analytics of dev first apis. (This user should be apim publisher user.)     | Optional → API level only      |

!!! note 
    -   WSO2 API Microgateway does not support the `x-auth-type` extension. Therefore, the following concepts of the auth types in WSO2 API Manager will not be work with the WSO2 API Microgateway.
        -   Application & Application User
        -   Application
        -   Application User
        -   None

>     -   However, if you want to invoke an API or a particular API resource without an access token, you can use the `x-wso2-disable-security` extension. Find more information about this extension from [here](https://mg.docs.wso2.com/en/latest/how-tos/security/api-authentication/disabling-security/#disabling-security).

Reviewer: ```suggestion
    -   However, if you want to expose API/resource without security, you can use the `x-wso2-disable-security` extension. Find more information about this extension from [here](https://mg.docs.wso2.com/en/latest/how-tos/security/api-authentication/disabling-security/#disabling-security).
```
